cloud/endagaweb/templates/dashboard/tower_detail/deregister.html

```diff
@@ -105,6 +127,41 @@ <h4 class='modal-title'>
     that cannot correctly process a deregistration request.
     Please contact support if you have questions
     or would like to upgrade your towers.
+    </div>
+     <div class='content col-xs-12 col-sm-6' style="height:100px;color: #428bca;">
+     <label display="block" width="100px">Forcefully Deregister</label>
+    <input style ="color:#428bca;"class="checkboxd" type="checkbox" id="myCheck" />
+
+   </div>
+
+<!-- Added towerId for   ForcefullyDeregister checkbox-->
+
+    <div id="towerId" class='content col-xs-12 col-sm-6' style="display:none">
+    <p>
```

Comment:
Please can you change this so that the text displayed 'Taking this action...' (from here to line 154), which is the same in both the forced and unforced situations, is only defined once within the template. That probably means pulling it out of the conditional block.